frontend/components/Documents/index.tsx

```diff
@@ -17,18 +17,14 @@ type Filter = {
 };
 
 const filters: Filter[] = [
-  {
-    title: 'HTM',
-    filter: (meetings: Meeting[]) => meetings.filter((meeting) => meeting.title.toUpperCase().includes('HTM')),
-  },
-  {
-    title: 'VTM',
-    filter: (meetings: Meeting[]) => meetings.filter((meeting) => meeting.title.toUpperCase().includes('VTM')),
-  },
   {
     title: 'Styrelsemöten',
     filter: (meetings: Meeting[]) => meetings.filter((meeting) => meeting.title.toUpperCase().match(/(S\d{2})/)),
   },
+  {
+    title: 'Sektionsmöten',
+    filter: (meetings: Meeting[]) => meetings.filter((meeting) => meeting.title.toUpperCase().includes('HTM') || meeting.title.toUpperCase().includes('VTM')),
+  },
```

Comment:
Do we want to group these? Maybe we do - I personally don't really have any strong preference either way.

Reply:
I think so, and the people from at least the board agreed. I see no reason to split them up.